draft-irtf-cfrg-hpke.md

```diff
@@ -633,15 +660,15 @@ This assurance is based on the assumption that
 `AuthDecap(enc, skR, pkS)` produces the correct KEM shared secret
 only if the encapsulated value `enc` was produced by
 `AuthEncap(pkR, skS)`, where `skS` is the private key corresponding
-to `pkS`.  In other words, only two people could have produced this
+to `pkS`.  In other words, only two entities could have produced this
 secret, so if the recipient is one, then the sender is the other
 with overwhelming probability.
 
 The primary differences from the base case are:
```

Comment:
Since there's only one difference, we could just have this as a paragraph: "The primary difference from the base case is that the calls to ..."

Reply:
👍

Reply:
I'll make this change after merging!